src/components/BaseSelect/BaseSelect.css.ts

```diff
@@ -20,6 +20,7 @@ export const listWrapperRecipe = recipe({
   base: [
     {
       borderRadius: listItemBorderRadius,
+      padding: "4px",
```

Comment:
**question**: why explicit pixel values instead of `vars.spacing`?

Reply:
Good catch - fixed 🙇🏻